Testscripts/Windows/VERIFY-DPDK-NFF-GO.ps1

```diff
@@ -22,9 +22,8 @@ function Main {
 		if (IsGreaterKernelVersion -actualKernelVersion $currentKernelVersion -detectedDistro $global:detectedDistro) {
 			Write-LogInfo "Confirmed Kernel version supported: $currentKernelVersion"
 		} else {
-			$msg = "Unsupported Kernel version: $currentKernelVersion"
```

Comment:
this code is never executed, as only debian distro are supported, see line 13

Reply:
When add 'Checking OVS DPDK compatibility' should see this point.
When a lower version tested but Debian distro, this code can be executed. 
And one question here, in future, only support Debian ones? If yes, I will update the code.